.github/workflows/deploy-oracle.yml

```
@@ -0,0 +1,60 @@
+# Commented out - not deploying to Oracle Cloud
```

actionlint (1.7.11)

[error] workflow is empty

(syntax-check)